agreement/security/ir.model.access.csv

```diff
@@ -1,3 +1,5 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_agreement_read,Read access on agreement to Employees,model_agreement,base.group_user,1,0,0,0
 access_agreement_full,Full access on agreement grp,model_agreement,base.group_no_one,1,1,1,1
+access_agreement_type_read,Read access on agreement type to Employees,model_agreement_type,base.group_user,1,0,0,0
+access_agreement_type_full,Full access on agreement type grp,model_agreement_type,base.group_no_one,1,1,1,1
```

Comment:
Why giving full access to developer mode? I think it should be to settings group.

Reply:
Ok. What groups can I use here??